index.js

```diff
@@ -45,6 +45,30 @@ function filterResponseForImportantEvents(allEventsFromFetch) {
     return arrayOfImportantEvents;
 }
 
+function filterForAuthorAssociation(events = [], association = 'OWNER') {
+    return events.reduce(function (acc, event) {
```

Comment:
I would expect to see this implemented using `.filter` instead of `.reduce`. Could you switch to use filter or is there a reason you chose reduce instead? Although it works, using reduce adds a lot more code complexity which makes it harder to read and understand.